frontend/src/metabase/static-viz/components/LineAreaBarChart/utils/series.tsx

```diff
@@ -0,0 +1,189 @@
+import _ from "underscore";
+import { merge } from "icepick";
+import { isNotNull } from "metabase/core/utils/array";
```

Comment:
This is what I changed since https://github.com/metabase/metabase/pull/26484 isn't backported.